amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresher.java

```diff
@@ -252,7 +292,35 @@ public boolean createLeaseTableIfNotExists(@NonNull final Long readCapacity, @No
     public boolean createLeaseTableIfNotExists() throws ProvisionedThroughputException, DependencyException {
         final CreateTableRequest request = createTableRequestBuilder().build();
 
-        return createTableIfNotExists(request);
+        boolean tableExists = createTableIfNotExists(request);
+
+        if (leaseTablePitrEnabled) {
+            enablePitr();
+            log.info("Enabled PITR on table {}", table);
+        }
+
+        return tableExists;
+    }
+
+    private void enablePitr() throws DependencyException {
+        final UpdateContinuousBackupsRequest request = UpdateContinuousBackupsRequest.builder()
+                .tableName(table)
+                .pointInTimeRecoverySpecification(PointInTimeRecoverySpecification.builder()
+                        .pointInTimeRecoveryEnabled(true)
+                        .build())
```

Comment:
https://sdk.amazonaws.com/java/api/2.0.0/software/amazon/awssdk/services/dynamodb/model/UpdateContinuousBackupsRequest.Builder.html#pointInTimeRecoverySpecification-java.util.function.Consumer-

Instead of passing in a PointInTimeRecoverySpecification, you can pass in a Consumer<PointInTimeRecoverySpecification.Builder> like so:



```suggestion
                .pointInTimeRecoverySpecification(builder->builder.pointInTimeRecoveryEnabled(true))
```

Reply:
Thank you for this shorthand, did not know about it. Updated